cmd/httpclient/main.go

```diff
@@ -62,35 +60,41 @@ func mainfunc() (err error) {
 	client := httpx.NewClient(handlers.StdoutHandler)
 	if *common.FlagHelp {
 		flag.CommandLine.SetOutput(os.Stdout)
-		fmt.Printf("Usage: httpclient [flags]\n")
+		fmt.Printf("Usage: httpclient [flags] <url>\n")
```

Comment:
Is this accurate? It seems to me `-url <url>` is the way to provide an URL to this tool

Reply:
Yes. There was something fun. I had tested ooni.io. The program had not received my argument and used the default address which was ooni.io. :))

Reply:
I added a default argument because in my view this command is to point people at how to use `netx` in a simple way _and_ to showcase what it emits.